### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by SupreemeGuy**

### CODE REVIEW

**Comment by WilliamVenner** [comment]
Some of the translation strings have been removed or reverted to English in your changes

**Comment by SupreemeGuy** [comment]
Updated now

File: i18n/de.json [hunk 1]
```diff
@@ -68,8 +68,8 @@
 		"image": "Image",
```
Comment (Luiggi33) [inline i18n/de.json:68]:
```suggestion
		"image": "Bild",
```

File: i18n/de.json [hunk 2]
```diff
@@ -68,8 +68,8 @@
...
 		"unknown": "Unknown ({extension})"
```
Comment (Luiggi33) [inline i18n/de.json:69]:
```suggestion
		"unknown": "Unbekannt ({extension})"
```

**Review by Luiggi33 — CHANGES_REQUESTED** [review]

**Comment by WilliamVenner** [comment]
@SupreemeGuy Please can you address Luiggi33's suggestions? Cheers :D

File: i18n/de.json [hunk 3]
```diff
@@ -19,14 +19,14 @@
 	},
 	"game": "Spiel",
 	"filesystem": "Dateisystem",
-	"github_star": "Star on GitHub",
+	"github_star": "Mit einem Stern auf GitHub bewerten",
```
[the remark this answers is not in the document]
Comment (SupreemeGuy) [reply i18n/de.json:22]: Würde schon sagen, dass Sterne auf GitHub als Bewertung funktionieren. Ist so ähnlich wie bei YouTube mit den Likes. Schau dir mal https://docs.github.com/en/get-started/exploring-projects-on-github/saving-repositories-with-stars an :)

[the remark this answers is not in the document]
Comment (Luiggi33) [reply i18n/de.json:22]: They literally talk about saving your favorite repos, not rating them. It's like saying youtubes playlist feature is not to save, categorise and even promote certained content, but to rate it.

**Review by Luiggi33 — APPROVED** [review]

**Pull request merged**